### PR TITLE
Fix zmon-agent application label

### PR DESCRIPTION
**Pull request by sbrauner-z**
Fix zmon-agent application label
Signed-off-by: Shery Brauner <shery.brauner@zalando.de>

### CODE REVIEW

**Comment by hjacobs** [comment]
:+1:

File: cluster/manifests/zmon-agent/deployment.yaml [hunk 1]
```diff
@@ -4,17 +4,17 @@ metadata:
...
   labels:
-    application: "zmon-agent"
+    application: "zmon-agent-core"
     version: "0.1-a56-zv2"
 spec:
   replicas: {{.ConfigItems.zmon_agent_replicas}}
   selector:
     matchLabels:
-      application: zmon-agent
+      application: zmon-agent-core
```
Comment (mikkeloscar) [inline cluster/manifests/zmon-agent/deployment.yaml:13]: This won't work. We need to delete the deployment first and re-apply.

Comment (nolar) [reply cluster/manifests/zmon-agent/deployment.yaml:13]: BTW, that would be totally fine if you "move" the zmon-agent app from our account to yours — if this is possible — thus, no re-deployment will be needed. Anyway, we use the team-specific zmon-blah for our needs, as I have recently found in our repo.

Comment (mikkeloscar) [reply cluster/manifests/zmon-agent/deployment.yaml:13]: @nolar thanks, then I propose we do that instead. Is that ok with you @sbrauner-z ?

Comment (sbrauner-z) [reply cluster/manifests/zmon-agent/deployment.yaml:13]: @mikkeloscar @nolar sounds good to me if this is actually possible.

Comment (nolar) [reply cluster/manifests/zmon-agent/deployment.yaml:13]: Oops. Just noticed it is a public GitHub. Removed the internal links & editing history.

**Comment by jrake-revelant** [comment]
Closing this in favour of changing the ownership.

**Pull request closed without merging**